firebase-perf/e2e-app/src/androidTest/java/com/google/firebase/testing/fireperf/InstrumentationTestUtil.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// You may obtain a copy of the License at
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.testing.fireperf;
+
+import com.google.android.datatransport.Priority;
+import com.google.android.datatransport.cct.CCTDestination;
+import com.google.android.datatransport.runtime.TransportRuntimeTesting;
+
+public class InstrumentationTestUtil {
+  /**
+   * Blocks calling thread until all of Fireperf's persisted events are sent by Firelog. Must be
+   * called after events have persisted.
+   */
+  static void flgForceUploadSync() {
+    TransportRuntimeTesting.forceUpload(CCTDestination.LEGACY_INSTANCE, Priority.DEFAULT);
```

Comment:
I think the Priority.DEFAULT is once in every 30 seconds. Can we use Priority.HIGHEST to make sure the events get dispatched immediately?

https://github.com/firebase/firebase-android-sdk/blob/master/transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/util/PriorityMapping.java#L29

Reply:
From the last Acore sync, my understanding from Vlad's recommendation to use this was that this Priority is the type of event that we want to force-send. For Fireperf, all events are sent using [`Event.ofData()`](https://github.com/firebase/firebase-android-sdk/blob/58941173b766325b670c76e9ea851feb696e2a23/firebase-perf/src/main/java/com/google/firebase/perf/transport/FlgTransport.java#L56), and that is always [`Priority.DEFAULT`](https://github.com/firebase/firebase-android-sdk/blob/58941173b766325b670c76e9ea851feb696e2a23/transport/transport-api/src/main/java/com/google/android/datatransport/Event.java#L33). @vkryachko for confirmation

Reply:
A nudge to this comment! Apart from this, rest of the changes look good to me.